src/main/java/frc/robot/subsystems/Vision.java

```diff
@@ -0,0 +1,86 @@
+package frc.robot.subsystems;
+
+import org.photonvision.PhotonCamera;
+import org.photonvision.targeting.PhotonPipelineResult;
+import org.photonvision.targeting.PhotonTrackedTarget;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Vision extends SubsystemBase
+{
+    private static Vision  _instance;
+    
+    public static Vision getInstance()
+    {
+        if (_instance == null) 
+        {
+            _instance = new Vision();
+        }
+       
+        return _instance;
+    }
+   
+    private final PhotonCamera _camera0 = new PhotonCamera("camera1");
+    private final PhotonCamera _camera1 = new PhotonCamera("camera2");
+    private PhotonPipelineResult _latestResult0 = new PhotonPipelineResult();
+    private PhotonPipelineResult _latestResult1 = new PhotonPipelineResult();
+    
+   
+    private Vision()
+    {
+
+    } 
+
+    public boolean hasTargets0()
+    {
+        return _latestResult0.hasTargets();
+    }
+    
+    public boolean hasTargets1()
+    {   
+        return _latestResult1.hasTargets();
+    }
+    
+  @Override
+    public void periodic()
+    {
+        _latestResult0 = _camera0.getLatestResult();
+        _latestResult1 = _camera1.getLatestResult();
+    }    
+    public double skew0()
```

Comment:
Rename these accordingly with the cameras so we know which camera is what

Reply:
Agreed again. 

Additionally, you can shorten all of these functions to one line. Using `skew0` as an example:

```java
public double skew0()
{
    return _latestResult0.getBestTarget().getSkew();
}
```